packages/windows/picotool/build-picotool.sh

```diff
@@ -21,7 +21,7 @@ if [ ${sdkVersion:0:1} -ge 2 ]; then
     cd pico-sdk/tools/pioasm
     mkdir -p build
     cd build
-    cmake -G Ninja .. -DCMAKE_BUILD_TYPE=Release -DPIOASM_FLAT_INSTALL=1 -Wno-dev
+    cmake -G Ninja .. -DCMAKE_BUILD_TYPE=Release -DPIOASM_FLAT_INSTALL=1 -DPIOASM_VERSION_STRING=$sdkVersion -Wno-dev
```

Comment:
Just out of curiosity: Why does this file use `$sdkVersion` when the other two use `$version` ?
I wonder if i might be worth making them consistent, to make future maintenance easier?